### ISSUE TITLE
Standalone Executable is too specific

### ISSUE THREAD

**Issue by stevespringett** [jowasp/review-features#3]
A standalone executable assumes that a command line or gui tool or utility is the deliverable. A web application such as OWASP Dependency-Track is delivered as a WAR. Similarly, the OWASP Java Encoder Project and OWASP ESAPI is delivered as a JAR for use in other applications. Neither project will produce an executable. The final delivery of being 'executable' is too specific.

**Comment by jowasp** [comment]
Agree. This will only play a role in certain kind of projects. We should remove this from the criteria.
